component/kafka/group/component.go

```diff
@@ -202,7 +202,7 @@ func (c *Component) processing(ctx context.Context) error {
 				}
 
 				// `Consume` should be called inside an infinite loop, when a
-				// server-side re-balance happens, the consumer session will need to be
+				// server-side rebalance happens, the consumer session will need to be
```

Comment:
This change slipped through :thinking: I was changing the wording because I noticed that Sarama always use `rebalance`, but I can remove it from this PR.